docs/src/components/NavbarItems/CurrentVersionNavbarItem.js

```diff
@@ -37,7 +34,7 @@ export default function CurrentVersionNavbarItem({ ...props }) {
           <div className={styles.versionWrapper}>
             <DefaultNavbarItem
               {...props}
-              label={`${latestVersion}`}
+              label={"edge"}
```

Comment:
Will this also affect the archived docs for specific OPA releases, or that's generated/created some other way?

Reply:
Yeah I kinda realised this was mostly not unhelpful unless we also had a way to set it. That will be used in the opa-docs repo, but this allows the site to be built with a version set.